NEWS

```diff
@@ -1,3 +1,25 @@
+etcd v3.1.0 (2017-01-13)
+- faster linearizable reads (implements Raft read-index)
+- automatic leadership transfer when leader steps down
+- etcd uses default host IP if advertise URL is not given
+- etcd --strict-reconfig-check flag is set by default
+- cluster rejects removing members if quorum will be lost
+- SRV records (e.g., infra1.example.com) must match the discovery domain
+  (i.e., example.com) if no custom certificate authority is given
+- discovery now has upper limit for waiting on retries
+- get responses are sorted in ascending order by default
+- add etcd --log-output flag
+- add etcd --metrics flag
+- add lease time-to-live API; fetch lease information
+- add v3 etcdctl get --print-value-only flag
+- add v3 etcdctl make-mirror --dest-prefix flag
+- add clientv3 Sync, SetEndpoints; update endpoints at runtime
+- drop clientv3.Config.Logger field
+- recipies now embed concurrency.Session
```

Comment:
@xiang90 It was for https://github.com/coreos/etcd/pull/6172.

Should we mention that? It's API change in `contrib/recipes`.
Maybe just `remove session manager and add ttl in recipes`?